src/test/js/config/runUITestsSauceLabs.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+ supportedBrowsers=`sed '/\/\//d' ./supportedBrowsers.js | sed '/: {/!d' | sed "s/[\'\:\{ ]//g"`
```

Comment:
Remove leading whitespace

Reply:
back ticks are not advised for this kind of assignment. Use `$(sed '/\/\//d' ...)` instead.

Reply:
This is also not written by me, but I will apply the changes as suggested